aztec-nargo/compile_then_postprocess.sh

```diff
@@ -40,7 +40,7 @@ for artifact in $artifacts_to_process; do
 
     echo "Generating verification key for function $fn_name"
     # BB outputs the verification key to stdout as raw bytes, however, we need to base64 encode it before storing it in the artifact
-    verification_key=$($BB write_vk_for_ivc -b ${fn_artifact_path} -o - | base64)
+    verification_key=$($BB write_vk --scheme client_ivc -b ${fn_artifact_path} -o - | base64)
```

Comment:
I guess this is immediately going to become confusing when we add a flow to write a CIVC VK. I wonder if this should just become the write_vk for mega honk..

Reply:
Yeah I thought about this but then it's the only mega honk command which seems weird. It think it makes sense to put it behind the --input_type flag in CIVC?